.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
 # Objective
 
-- Describe the objective or issue this PR addresses.
-- If you're fixing a specific issue, say "Fixes #X".
+Why are you making these changes?
+Link any associated issues with "Fixes #X" so they can be automatically closed.
+If this is a particularly complex PR, please add a clear list of the changes made.
```

Comment:
```suggestion
If this is a complex Pull Request, add a list of the changes made.
```
Highly opinionated feedback:
- Avoid words not adding precise meaning. (“complex” is already subjective, so “particularly complex” may add different meaning to different persons)
- “please” adds to the feeling of welcomeness, but why here especially ? It sounds like it was added mainly because there is an “if”..?
- Avoid abbreviations

Reply:
I don’t mind much the “PR” (Pull Request) abbreviation, but I like when all abbreviations are introduced explicitly at least the first time the reader encounters it. No exceptions anymore, too often I’ve struggled with them 😂.

Reply:
I don't think the "please" necessarily needs to be removed. I welcome the welcomeness haha

Reply:
```suggestion
If this is a complex PR, please add a list of the changes made.
```